examples/block_jid.py

```diff
@@ -114,6 +116,11 @@ def jid(s):
             help="If given, prints the block list at the end of the operation",
         )
 
+        config_default = os.path.expanduser(
+            '~/.config/aioxmpp_credentials.ini')
```

Comment:
I would prefer using the current working directory or the directory the examples are located in. Using ~/.config is not portable (we would have to inspect XDG_CONFIG_HOME on linuxes and who-knows-what on other OSes) and using XDG or appdirs is overkill here.